src/Apps/Sell/Routes/ArtistNotEligibleRoute.tsx

```diff
@@ -20,6 +24,32 @@ interface ArtistNotEligibleRouteProps {
 
 export const ArtistNotEligibleRoute: React.FC<ArtistNotEligibleRouteProps> = props => {
   const artist = useFragment(FRAGMENT, props.artist)
+  const { showAuthDialog } = useAuthDialog()
+  const { router } = useRouter()
+  const { isLoggedIn } = useSystemContext()
+
+  const onAddToMyCollection = () => {
+    if (isLoggedIn) {
+      router.push("/collector-profile/my-collection/artworks/new")
+    } else {
+      showAuthDialog({
+        mode: "SignUp",
+        options: {
+          title: mode =>
+            mode === "Login"
+              ? "Log in to Add to My Collection"
+              : "Sign up to Add to My Collection",
```

Comment:
That's the pattern we use in other auth dialogs ([examples](https://github.com/search?q=repo%3Aartsy%2Fforce+%22log+in+to%22&type=code)).
```suggestion
              ? "Log in to add to My Collection"
              : "Sign up to add to My Collection",
```